### ISSUE TITLE
TypeError: getShop() returning null

### ISSUE THREAD

**Issue by geenious** [craftcms/shopify#56]
### Description
During initial setup after entering the 4 shopify variables in settings, the webhooks navigation item appears. When visiting that url (/admin/shopify/webhooks?site=default) a Type Error is returned. I'm sure I am doing something foolish.

```
TypeError
Shopify\Auth\Session::getShop(): Return value must be of type string, null returned
```

### Steps to reproduce

1. Add/Install Shopify plugin
2. Enter required variable settings and save
3. Click 'Webhooks'

### Additional info

- Craft version: 4.3.6.1
- PHP version: 8.0.26
- Database driver & version: MySQL 10.3.37
- Plugins & versions: Shopify | 3.1.0

**Comment by geenious** [comment]
I've got this working now. The issue was the $SHOPIFY_HOSTNAME var. The documentation references finding the hostname from the store settings screen here: https://github.com/craftcms/shopify/blob/develop/docs/shopify-hostname.png

The store I'm working with had a custom domain setup like store.{company-name}.com which I used and gave me the TypeError. I changed the hostname to the more shopify URL of {store-name}.myshopify.com and we're in business.
